wallet/src/Pos/Wallet/Web/Account.hs

```diff
@@ -62,12 +62,13 @@ getSKById wid = do
 
 getSKByAccAddr
     :: AccountMode ctx m
-    => PassPhrase
+    => IsBootstrapEraAddr
+    -> PassPhrase
     -> CWAddressMeta
     -> m EncryptedSecretKey
-getSKByAccAddr passphrase addrMeta@CWAddressMeta {..} = do
+getSKByAccAddr ibe passphrase addrMeta@CWAddressMeta {..} = do
     (addr, accKey) <-
-        deriveAccountSK passphrase (addrMetaToAccount addrMeta) cwamAccountIndex
+        deriveAccountSK ibe passphrase (addrMetaToAccount addrMeta) cwamAccountIndex
```

Comment:
Actually there are some places where we assume the only way to build an address by given secret key.
One of such places is just below this line.

There may be more caveats like this one. Please, try to emulate switching to another era (switching hardcoded value while nodes are off should be enough), whether, say, transaction creation will still work. I'll accept after that